src/core/env/prod/backend.tfvars

```diff
@@ -1,4 +1,4 @@
 resource_group_name  = "io-infra-rg"
-storage_account_name = "pagopainfraterraformprod"
+storage_account_name = "selcpstinfraterraform"
 container_name       = "azureadstate"
 key                  = "prod.terraform.tfstate"
```

Comment:
Should be **terraform.tfstate** for all (we have different container and storage account)

Reply:
Actual resource name used for UAT is selcustinfraterraform, as created during authorizing process.
Should we first try to manually rename it inside the subscription?

Reply:
Sorry it's for key